main.go

```diff
@@ -399,6 +405,10 @@ func (c *config) addTags(fieldName string, tags *structtag.Tags) (*structtag.Tag
 		unknown = true
 	}
 
+	if c.valueFormat != "" {
+		name = strings.ReplaceAll(c.valueFormat, "$value", name)
```

Comment:
What do you think of renaming `$value` to `$field`? Because this will be used to create a new value, but it indicates the field name. I think it's better and easier that way.

Reply:
I agree, it sounds more representative.